autoload/neomake.vim

```diff
@@ -737,15 +770,40 @@ function! s:GetTabWinForJob(job_id) abort
 endfunction
 
 function! s:RegisterJobOutput(jobinfo, lines, source) abort
-    let lines = copy(a:lines)
     let maker = a:jobinfo.maker
 
     if !get(maker, 'file_mode')
-        call s:ProcessJobOutput(a:jobinfo, lines, a:source)
-        call neomake#signs#PlaceVisibleSigns()
+        if s:CanProcessJobOutput()
+            call s:ProcessJobOutput(a:jobinfo, a:lines, a:source)
+            call neomake#signs#PlaceVisibleSigns()
```

Comment:
I've noticed that `call neomake#highlights#ShowHighlights()` might be missing here?!
/cc @paretje (via ca87544)